dictionaries/CallMap.php

```diff
@@ -359,29 +359,29 @@
-'array_diff_uassoc\'1' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest='=>'array|callable(mixed,mixed):int'],
-'array_diff_ukey' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'key_comp_func'=>'callable(mixed,mixed):int'],
-'array_diff_ukey\'1' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest='=>'array|callable(mixed,mixed):int'],
+'array_combine' => ['array', 'keys'=>'string[]|int[]', 'values'=>'array'],
+'array_count_values' => ['array<int>', 'array'=>'array<string|int>'],
+'array_diff' => ['array', 'array'=>'array', '...arrays='=>'array'],
+'array_diff_assoc' => ['array', 'array'=>'array', '...arrays='=>'array'],
+'array_diff_key' => ['array', 'array'=>'array', '...arrays='=>'array'],
+'array_diff_uassoc' => ['array', 'array'=>'array', 'rest'=>'array', 'data_comp_func'=>'callable(mixed,mixed):int'],
+'array_diff_uassoc\'1' => ['array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest='=>'array|callable(mixed,mixed):int'],
+'array_diff_ukey' => ['array', 'array'=>'array', 'rest'=>'array', 'key_comp_func'=>'callable(mixed,mixed):int'],
+'array_diff_ukey\'1' => ['array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest='=>'array|callable(mixed,mixed):int'],
 'array_fill' => ['array<int,mixed>', 'start_index'=>'int', 'count'=>'int', 'value'=>'mixed'],
 'array_fill_keys' => ['array', 'keys'=>'array', 'value'=>'mixed'],
-'array_filter' => ['associative-array', 'array'=>'array', 'callback='=>'callable(mixed,mixed=):scalar|null', 'mode='=>'int'],
-'array_flip' => ['associative-array<mixed,int>|associative-array<mixed,string>', 'array'=>'array'],
-'array_intersect' => ['associative-array', 'array'=>'array', '...arrays='=>'array'],
-'array_intersect_assoc' => ['associative-array', 'array'=>'array', '...arrays='=>'array'],
-'array_intersect_key' => ['associative-array', 'array'=>'array', '...arrays='=>'array'],
-'array_intersect_uassoc' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'key_compare_func'=>'callable(mixed,mixed):int'],
-'array_intersect_uassoc\'1' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest'=>'array|callable(mixed,mixed):int'],
-'array_intersect_ukey' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'key_compare_func'=>'callable(mixed,mixed):int'],
-'array_intersect_ukey\'1' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest'=>'array|callable(mixed,mixed):int'],
+'array_filter' => ['array', 'array'=>'array', 'callback='=>'callable(mixed,mixed=):scalar|null', 'mode='=>'int'],
+'array_flip' => ['array<string|int>', 'array'=>'array<string|int>'],
```

Comment:
The value must be string or int not the entire array. The input array has the same restriction.